actors/miner/src/lib.rs

```diff
@@ -1203,11 +1210,10 @@ impl Actor {
         // Group declarations by deadline
```

Comment:
Delete the call to request_deal_data at line 1186. We don't need the results any more.